kuksa-client/kuksa_client/grpc/aio.py

```diff
@@ -55,6 +56,7 @@ class VSSClient(BaseVSSClient):
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self.channel = None
+        self.channel2 = None
```

Comment:
Should we possibly call them `channel_v1` and  `channel_v2`. To make sure that that the number refers to a version and not just arbitrary channels